DirectProgramming/DPC++FPGA/Tutorials/Features/private_copies/README.md

```diff
@@ -150,7 +150,10 @@ You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the Vi
 ## Examining the Reports
 Locate `report.html` in the `private_copies_report.prj/reports/` or `private_copies_s10_pac_report.prj/reports/` directory. Open the report in any of Chrome*, Firefox*, Edge*, or Internet Explorer*.
 
-On the main report page, scroll down to the section titled "Estimated Resource Usage". Each kernel name ends in the `private_copies` attribute argument used for that kernel, e.g., `kernelCompute1` uses a `private_copies` attribute value of 1. You can verify that the number of RAMs used for each kernel increases with the private_copies value used, with the exception of private_copies 0. Using private_copies 0 instructs the compiler to choose a default value, which is often close to the value that would give you maximum throughput.
+On the main report page, scroll down to the section titled "Estimated Resource Usage". Each kernel name ends in the `private_copies` attribute argument used for that kernel, e.g., `kernelCompute1` uses a `private_copies` attribute value of 1. You can verify that the number of RAMs used for each kernel increases with the `private_copies` value used, with the exception of `private_copies` 0. Using `private_copies` 0 instructs the compiler to choose a default value, which is often close to the value that would give you maximum throughput.
+
+### Analytically determining the `private_copies` value
+Once you have the main report page open, click on the dropdown titled "Throughput Analysis" and choose the "Loop Analysis" option. Now you can choose to look at "Kernel: Kernel<0>" and this will provide information for the three loops of Kernel<0>. Specifically, you want to be looking at the concurrency value found by clicking on the array `a` in the "Bottlenecks" section of the report. In the details pane it will now highlight that the concurrency value for `a` is 3 in addition to the fact that there is a throughput bottleneck on Loop "B3" that has a latency of 222.0. Using this information combined with the knowledge that the "Block Scheduled fMAX" is 240.00, we can make an analytical estimate that using `private_copies` of 3 or higher will deliver optimal throughput performance. 
```

Comment:
I don't understand why the block schedule fmax or the latency are relevant to determining the optimal number of private copies?